libtinysmb/smb_devoptab.c

```diff
@@ -1456,7 +1456,7 @@ bool smbInitDevice(const char* name, const char *user, const char *password, con
 {
 	int i;
 
-	if(!name || strlen(name) > 9)
+	if(!name || strlen(name) > 8)
```

Comment:
funny enough, this might also fix bugs/memory corruption if the user would pass a string = 9 characters